### PR TITLE
R: updates for packages

### PR DESCRIPTION
**Pull request by barracuda156**
#### Description

Finally update `R-nebula` (required a lot of new dependencies), fix `R-adjclut` for clangs and `R-duckdb` for 10.7–10.11.
Add some testing support.

###### Type(s)

- [x] bugfix
- [x] enhancement
- [ ] security fix

###### Tested on

macOS 10.6
Xcode 3.2

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [x] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL in commit message? 
- [ ] checked your Portfile with `port lint --nitpick`?
- [x] tried existing tests with `sudo port test`? (where tests are added and for new ports)
- [ ] tried a full install with `sudo port -vst install`?
- [ ] tested basic functionality of all binary files?
- [ ] checked that the Portfile's most important [variants](https://trac.macports.org/wiki/Variants) haven't been broken?

### CODE REVIEW

File: R/R-duckdb/Portfile [hunk 1]
```diff
@@ -22,10 +26,28 @@ compiler.cxx_standard 2017
...
+    post-patch {
+        # https://github.com/duckdb/duckdb/issues/7009
+        if {[string match macports-gcc* ${configure.compiler}] && (${configure.build_arch} in [list i386 ppc])} {
+            reinplace "s|@ATOMIC@|-latomic|" ${worksrcpath}/src/Makevars
+        } else {
+            reinplace "s|@ATOMIC@||" ${worksrcpath}/src/Makevars
+        }
+
+        # https://trac.macports.org/ticket/69902
+        # On 10.6 and earlier LegacySupport is already used,
```
Comment (ryandesign) [inline R/R-duckdb/Portfile:41]: What causes legacysupport to be used on 10.6 and earlier but not later? I'm looking specifically at the line `legacysupport.newest_darwin_requires_legacy 15` above.

Comment (barracuda156) [reply R/R-duckdb/Portfile:41]: @ryandesign `R` portfile sets `legacysupport.newest_darwin_requires_legacy 10`, so for 10.6 and below flags are baked into `R`, it seems. On my system there are used for every port automatically, and apparently on buildbots it is the same.

Comment (barracuda156) [reply R/R-duckdb/Portfile:41]: I guess we do not want to enable it via `R` portfile for everything just to fix a single port. So I mark it as resolved.

**Comment by barracuda156** [comment]
@herbygillot @ryandesign This should be good to merge.

**Pull request merged**